spark/src/main/java/org/opensearch/sql/spark/dispatcher/InteractiveQueryHandler.java

```diff
@@ -94,7 +93,7 @@ public DispatchQueryResponse submit(
       session =
           sessionManager.createSession(
               new CreateSessionRequest(
-                  jobName,
+                  clusterName,
```

Comment:
What is the benefit of transferring the jobname formatting to the callee side? I'd rather leave them at handler level for consistency.

Reply:
I tried to do that but it requires a lot of refactoring and the current code structure is little convoluted. But I will take a look at it again.